metricflow/test/fixtures/model_yamls/simple_model/metrics.yaml

```diff
@@ -4,182 +4,160 @@ metric:
   description: "bookings metric"
   type: simple
   type_params:
-    measures:
-      - bookings
+    measure: bookings
```

Comment:
I guess this is getting wrapped by the DSI property accessor or something? Or we're populating input_measures from it?